internal/pkg/manifest/backend_svc.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	backendSvcManifestPath = "services/backend/manifest.yml"
+	backendSvcManifestPath = "workloads/backend/manifest.yml"
```

Comment:
I feel like it still makes sense to have `services` somewhere in the path what do you think of this?
```suggestion
	backendSvcManifestPath = "workloads/services/backend/manifest.yml"
```
And the common files between workloads can go to: `"workloads/common/param.json.tmpl"`